src/components/ui/Mark.tsx

```diff
@@ -0,0 +1,70 @@
+import classnames, {
+  alignItems,
+  borderColor,
+  borderRadius,
+  borderWidth,
+  boxSizing,
+  cursor,
+  display,
+  fontFamily,
+  fontSize,
+  height,
+  justifyContent,
+  width,
+} from 'classnames/tailwind'
+
+const questionStyles = (size?: Sizes, disabled?: boolean) =>
+  classnames(
+    display('flex'),
+    justifyContent('justify-center'),
+    alignItems('items-center'),
+    fontSize(size === Sizes.Small ? 'text-xs' : 'text-base'),
```

Comment:
`base` is default, use `{'text-xs': Sizes.Small}`

Reply:
ty